contentcuration/contentcuration/frontend/channelList/views/Channel/StudioMyChannels.vue

```diff
@@ -0,0 +1,533 @@
+            orientation: 'vertical',
+            count: 3,
+          },
+        ]"
+        class="cards"
+      >
+        <KCard
+          v-for="(channel, index) in listChannels"
+          :key="channel.id"
+          :headingLevel="2"
+          thumbnailDisplay="small"
+          :thumbnailSrc="thumbnailSrc(channel)"
+          :thumbnailAlign="'left'"
+          :thumbnailScaleType="'fitXY'"
+          :orientation="windowIsSmall ? 'vertical' : 'horizontal'"
+          :title="channel.name"
+          :titleMaxLines="2"
+          :data-testid="`card-${index}`"
+          @click="goToChannelRoute(channel)"
+        >
+          <template #thumbnailPlaceholder>
+            <KIcon
+              :style="{ fontSize: '48px' }"
+              icon="image"
+            />
+          </template>
+          <template #belowTitle>
+            <div class="cards-title">
+              <div class="cards-resource">
+                <span> {{ $tr('resourceCount', { count: channel.count || 0 }) }} </span>
+                <span>
+                  {{ language(channel) }}
+                </span>
+              </div>
+              <div
+                class="cards-desc"
+                :style="{ color: $themePalette.black }"
+              >
+                {{ channel.description }}
+              </div>
+            </div>
+          </template>
+          <template #footer>
+            <div class="footer">
+              <div class="footer-left">
+                <span :style="{ color: $themePalette.grey.v_700 }">
+                  {{
+                    channel.last_published
+                      ? $tr('lastPublished', {
+                        last_published: $formatRelative(channel.last_published, {
+                          now: new Date(),
+                        }),
+                      })
+                      : $tr('unpublishedText')
+                  }}
+                </span>
+                <div>
+                  <KTooltip
+                    :reference="`lastUpdatedTime-${index}`"
+                    placement="bottom"
+                    :refs="$refs"
+                  >
+                    {{
+                      $tr('lastUpdated', {
+                        updated: $formatRelative(channel.modified, { now: new Date() }),
+                      })
+                    }}
+                  </KTooltip>
+                  <KIcon
+                    v-if="hasUnpublishedChanges(channel)"
+                    :ref="`lastUpdatedTime-${index}`"
+                    :color="$themePalette.green.v_600"
+                    icon="dot"
+                  />
+                </div>
+              </div>
+              <div class="footer-right">
+                <router-link
+                  :data-testid="`details-button-${index}`"
+                  :to="channelDetailsLink(channel)"
+                  @click.native.stop
+                >
+                  <KIconButton
```

Comment:
`KIconButton` shouldn't be used within router link. This creates an invalid markup structure

```
<a><button></a>
```

and causes few problems with a11y, one of them being keyboard navigation (try TAB key which now displays two focus ring in sequence for the same thing - only one should be there)

Can you follow [this example](https://design-system.learningequality.org/kcard#interactive-elements)? Use this structure:

```
<router-link>
  <KIcon>
</router-link>
<KTooltip> // reference `KIcon`
```